content/blog/2024-08-02-unikraft-gsoc-lxboot_x86.mdx

```diff
@@ -0,0 +1,106 @@
+---
+title: "GSoC'24: Linux x86 Boot Protocol Support"
+description: |
+  The Linux boot protocol plays an important role in the initialization of the Linux operating system, emphasizing the importance of system optimization and scalability.
+publishedDate: 2024-08-02
+image: /images/unikraft-gsoc24.png
+tags:
+- gsoc
+- gsoc24
+- booting
+authors:
+- Mihnea Firoiu
+---
+
+## Setup in Unikraft
+
+First of all, I added the following lines to the [lxboot.S](https://github.com/unikraft/unikraft/blob/staging/plat/kvm/x86/lxboot.S) file from Unikraft:
```

Comment:
Don't link to staging/master branches, link to specific set in stone links, like release tags, certain commits, etc.
The staging/master branches will change in the future, so the links might (more than likely) break.